### DISCUSSION TITLE
Accessing property inside filter functions not possible?

### DISCUSSION THREAD

**Discussion by WojciechRhdt** [rappasoft/laravel-livewire-tables#1038] [Q&A] [answered]
I'm trying to access a property inside the filter function, but allways get null as response. I'm using the mount function to initiate the property like this:

`    public function mount() {
        $this->papertopics = PaperTopic::select(['id', 'title'])->pluck('title', 'id')->toArray();
    }
`
When I access the property for example inside the configure or columns functions it is working fine. But if I try to acces the property inside the filter function I just get a null as result. Am i doing anything wrong?

```
public function columns(): array {
// $this->papertopics outputs the right model
}

public function filters(): array {
// $this->papertopics outputs null
}
```

**Comment by lrljoe** [comment, marked as answer]
From memory, filters is run before anything else.  So you'll need to make sure you're setting your variable in the right place.

**Reply by WojciechRhdt** [reply]
I'm using the boot function right now ans this works fine. Thanks!
